tests/unit/__init__.py

```diff
@@ -1 +1,10 @@
- 
+# -*- coding: utf-8 -*-
```

Comment:
We need that file because there are several imports inside unit tests.
E.g.  in the `tests/unit/test_alpha.py`: 
```python
from .responses import fake_response_from_file
```